### ISSUE TITLE
Why http/1.0 is forced?

### ISSUE THREAD

**Issue by 23doors** [mattn/go-uwsgi#4]
Any particular reason why http/1.0 is forced here: 
https://github.com/mattn/go-uwsgi/blob/master/uwsgi.go#L251 and here https://github.com/mattn/go-uwsgi/blob/master/uwsgi.go#L273 ?

**Comment by mattn** [comment]
I'm not sure, I don't remember, but maybe, when I tested this, the server did not return a response. This is workaround.

**Comment by 23doors** [comment]
Tried removing these 2 lines and everything seemed to be working fine at least for http/1.1 but I haven't done any extensive tests. And I am running it behind a uwsgi (internal routing) which may for some reason yield different results than running it behind nginx + uwsgi pass.

**Comment by mattn** [comment]
If you solve your issue, please send me p-r.

**Comment by 23doors** [comment]
https://github.com/mattn/go-uwsgi/pull/5

**Comment by 23doors** [comment]
Any chance you could take a look? If you don't have time right now, I'll just use my fork for the time being

**Comment by mattn** [comment]
Test fails. Could you handle this? https://travis-ci.org/mattn/go-uwsgi

**Comment by 23doors** [comment]
Made a fix for regression: https://github.com/mattn/go-uwsgi/pull/6 but actually it seems that tests were failing for a long time now. Fixed the error, but tests seem to hang anyway. I tested version before PR - and it also hangs on me so it doesn't seem to be a regression. Sadly, I don't have time right now to debug what is wrong with the testcase especially as it looks unfinished.

Anyway. TestBasic should work after this PR.
TestServer is another story though.
From what I gathered:
```
passenger := &Passenger{"tcp", "127.0.0.1"}
```
this is wrong, no port in addr - and server that it points to is not running

```
l, err := net.Listen("tcp", "127.0.0.1:0")
```
listener is declared but unused. And yet we later on try to connect to it which causes timeout.
